docs/es/GIT.es.md

```diff
@@ -0,0 +1,43 @@
+# USO DE GIT:
+
+Hay algunas reglas que se aplican al interactuar con el repositorio del proyecto. Esto nos permitirá tener un estándar que cualquier desarrollador pueda entender. Es importante cumplir con estas reglas para que la herramienta de linting nos permita usar el repositorio.
+
+**Rama:**
+
+Cada vez que se le asigna un problema, debe crear una nueva sucursal. Esto se puede hacer a través del comando:
+
+```
+git branch <nombre de la rama>
+```
+
+Por conveniencia, las sucursales deben tener una estructura definida. Por ejemplo, si el issue se trata de agregar una nueva función, la rama debería ser:
+
+```
+feature/#número-de-issue-breve-descripción-de-la-función
+```
+
+Supongamos que tenemos el issue n. ° 12 que nos pide que agreguemos un encabezado a la página principal. Entonces, la rama quedaría de la siguiente manera:
+
+```
+feature/#12-añadir-encabezado-a-la-página-principal
+```
+
+Si se trata de solucionar un bug, solo tenemos que anteponer la palabra "fix". Por ejemplo:
+
+```
+fix/#14-remove-bad-media-queries-in-rating-component
+```
+
+**Compromisos:**
```

Comment:
"Compromisos" can sound weird hehe. What do you think of change it for "Confirmaciones" or "commits"